test/functional/lua/hl_spec.lua

```diff
@@ -214,4 +214,38 @@ describe('vim.hl.on_yank', function()
       vim.hl.range(0, ns, 'Search', { 0, 0 }, { 0, 0 }, { timeout = 0 })
     end)
   end)
+
+  it("highlight last character when do 'exclusive' motion", function()
+    clear()
```

Comment:
`clear()` isn't needed. `before_each()` already does that.
```suggestion
  it('highlight last character with exclusive motion', function()
```